tutorial.md

```diff
@@ -0,0 +1,234 @@
+# Dynamico Tutorial
+
+In this tutorial you're going to use a demo registry we've set up to host and serve components. The goal here is to try out the work flow with dynamico, and exprience how easy it is to integrate and use dynamico.
+
+We are going to create a new react app and integrate dynamico client into it. After that we're going to create a new component, developing it locally and publish it to a server and see it work in production mode. When we're finished with that we will update the component and see the changes applied in your app.
+
+This guide assumes you have a node version compatible with `create-react-app` (node 10 works fine), yarn and npx available.
+
+## Host App
+
+Let's start with creating an app:
+```bash
+$ npx create-react-app dynamico-demo
+```
+
+Now install the dependencies:
+```bash
+$ cd dynamico-demo
+& yarn add @dynamico/core @dynamico/react
+```
+
+Open `App.js` and add these import statements to the top of the file:
+```javascript
+import { DynamicoClient } from '@dynamico/core';
+import { DynamicoProvider } from '@dynamico/react';
+```
+
+Here we import both dynamico client which is the client side 
+library responsible for extracting components from the code 
+and dynamico provider which is used to pass the client along on the context to so you can use dynamic components just like a regular react component.
+
+We're also going to need ReactDOM available in this scope so let's go ahead and import that too:
```

Comment:
why?

Reply:
it's one of the default dependencies we add in plop when creating a component... so for this to Just Work ™️  you have to add react-dom. We can add moment or lodash for the update step in the tutorial.

Reply:
We should definitely remove it from there :X